### ISSUE TITLE
Sender for package is not required

### ISSUE THREAD

**Issue by JanGalek** [Salamek/PplMyApi#20]
Sender for package is not required, but constructor of Package need it. @Salamek Do you want I send PR ?

**Comment by Salamek** [comment]
@JanGalek well we have problem here,

```php
Sender $sender,
Recipient $recipient,
SpecialDelivery $specialDelivery = null,
```
After Sender there is another required parameter... that means we need to change order of parameters to:
```php
Recipient $recipient,
Sender $sender = null,
SpecialDelivery $specialDelivery = null,
```

That is BC breaking change...
And Label generator needs Sender to be set... (or sender is also not required to be printed on label ?)

PS: This project was originaly build on top of "beta" documentation and this stuff wasnt there yet...

**Comment by JanGalek** [comment]
@Salamek We work with CreatePackages only.

PPL support say that, someting they have problem when they return package whitch have sender. Its problem of PPL.

Maybe do this change for version 2.x.x ?

**Comment by Salamek** [comment]
This is closed by PR #21
